### DISCUSSION TITLE
LDAP config missing attributes

### DISCUSSION THREAD

**Discussion by coreymr** [grafana/grafana#44917] [General]
So I've configured LDAP for my Grafana dashboard, however, the LDAP Attributes with my User Information are not getting populated.  

LDAP lookup works, since it finds my account and security LDAP group assignments; however, no information is populated.  So when I go and look at the Users section in grafana, I have blank line items.

Any ideas?

v8.3.4

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
